applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_axisymmetric_element.hpp

```diff
@@ -30,21 +30,22 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) SmallStrainUPwDiffOrderAxisymmetricE
 public:
     KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(SmallStrainUPwDiffOrderAxisymmetricElement);
 
-    typedef std::size_t                         IndexType;
-    typedef Properties                          PropertiesType;
-    typedef Node                                NodeType;
-    typedef Geometry<NodeType>                  GeometryType;
-    typedef Geometry<NodeType>::PointsArrayType NodesArrayType;
-    typedef Vector                              VectorType;
-    typedef Matrix                              MatrixType;
+    using IndexType      = std::size_t;
+    using PropertiesType = Properties;
+    using NodeType       = Node;
+    using GeometryType   = Geometry<NodeType>;
+    using NodesArrayType = Geometry<NodeType>::PointsArrayType;
+    using VectorType     = Vector;
+    using MatrixType     = Matrix;
     /// The definition of the sizetype
-    typedef std::size_t SizeType;
+    using SizeType = std::size_t;
     using SmallStrainUPwDiffOrderElement::mConstitutiveLawVector;
+    using typename SmallStrainUPwDiffOrderElement::ElementVariables;
 
     ///----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
     /// Default Constructor
-    SmallStrainUPwDiffOrderAxisymmetricElement() : SmallStrainUPwDiffOrderElement() {}
+    using SmallStrainUPwDiffOrderElement::SmallStrainUPwDiffOrderElement;
```

Comment:
I'm sorry, but I don't follow what the intent of this change is. Could you please elaborate on it?

Reply:
This was a suggestion by sonarcloud ([code smell](https://sonarcloud.io/project/issues?fileUuids=AX3ANmpRpP8AcatU8LhR&resolved=false&types=CODE_SMELL&id=KratosGeomechanics&open=AX3ANm-apP8AcatU8MEE&tab=why)). Since we don't do any extra initialization, it seems you can include this 'using' statement instead of redefining an empty one.